src/Data/PlaceCal/Partners.elm

```diff
@@ -125,6 +128,7 @@ decodePartner =
         |> OptimizedDecoder.Pipeline.optional "address" (OptimizedDecoder.map Just addressDecoder) Nothing
         |> OptimizedDecoder.Pipeline.required "areasServed" (OptimizedDecoder.list serviceAreaDecoder)
         |> OptimizedDecoder.Pipeline.optionalAt [ "address", "geo" ] (OptimizedDecoder.map Just geoDecoder) Nothing
+        |> OptimizedDecoder.Pipeline.optional "logo" (OptimizedDecoder.map Just OptimizedDecoder.string) Nothing
```

Comment:
Did you have a look at the data? I think last time I checked there was maybe one logo in there.

Reply:
I thnk that's what led me to make this a very low priority, but @kimadactyl keen to get it in. I agree too that the partners will see value in it (if they know it's a thing they can use)

Reply:
im not sure what the question for me is here sorry?

Reply:
The question is do you want us to implement this even if only 1 or 2 partners have logos?

Reply:
i think logos arent being added because they don't show up - so yes?